utilities/decrypt_data/decrypt_multi.py

```
@@ -0,0 +1,237 @@
+ch = logging.StreamHandler()
+ch.setFormatter(formatter)
+logger.addHandler(ch)
+# File handler
+fh = logging.FileHandler(log_filename, mode="a")
+fh.setFormatter(formatter)
+logger.addHandler(fh)
+
+
+def decrypt_values(encrypted_fields: dict) -> dict:
+    """
+    Send only non‐empty fields to the decrypt API,
```

Ruff (0.12.2)

Docstring contains ambiguous `‐` (HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)